ledger/sandbox-on-x/BUILD.bazel

```diff
@@ -529,3 +538,141 @@ conformance_test(
         "--verbose",
     ],
 )
+
+SERVERS_HOCON = {
+    "h2database": {
+        "binary": ":app-hocon",
+    },
+    "postgresql": {
+        "binary": ":conformance-test-postgres-bin-hocon",
+    },
+    "oracle": {
+        "binary": ":conformance-test-oracle-bin-hocon",
+        "tags": oracle_tags,
+    },
+}
+
+da_scala_binary(
+    name = "app-hocon",
+    main_class = "com.daml.ledger.sandbox.MainHocon",
+    tags = [
+        "fat_jar",
+        "maven_coordinates=com.daml:sandbox-on-x-app-jar:__VERSION__",
+        "no_scala_version_suffix",
+    ],
+    visibility = ["//visibility:public"],
+    runtime_deps = [
+        "@maven//:ch_qos_logback_logback_classic",
+        "@maven//:ch_qos_logback_logback_core",
+        "@maven//:com_h2database_h2",
+    ],
+    deps = [
+        ":sandbox-on-x-app-lib",
+    ],
+)
+
+da_scala_binary(
+    name = "conformance-test-oracle-bin-hocon",
+    main_class = "com.daml.ledger.sandbox.MainWithEphemeralOracleUserHocon",
+    visibility = ["//visibility:public"],
+    runtime_deps = [],
+    deps = [
+        ":sandbox-on-x-test-app",
+    ],
+)
+
+da_scala_binary(
+    name = "conformance-test-postgres-bin-hocon",
+    main_class = "com.daml.ledger.sandbox.MainWithEphemeralPostgresqlHocon",
+    visibility = ["//visibility:public"],
+    runtime_deps = [],
+    deps = [
+        ":sandbox-on-x-test-app",
+    ],
+)
+
+#Duplicated tests using new hocon-style configuration
+conformance_test(
```

Comment:
How many of these conformance tests get run on each PR to main.  How long does it add to the build ?

Reply:
I have added them to make sure HOCON configuration works just like old CLI. In fact did not plan them to be in parallel and they should have been replacing old cli implementation. Let's agree first on the sequence of work merged to `main` and deprecation policy, it would be then clear what to do with the tests.